foundation/org.eclipse.persistence.oracle/src/org/eclipse/persistence/platform/database/oracle/Oracle19Platform.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0,
+ * or the Eclipse Distribution License v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+ */
+
+// Contributors:
+//     IBM - initial drop
+//     09/19/2019 - Lalitha Ananda
+//       - 548814 : EL not Auto Detecting Oracle 19C
+
+package org.eclipse.persistence.platform.database.oracle;
+
+public class Oracle19Platform extends Oracle12Platform {
```

Comment:
I think there shoud be `extends Oracle18Platform`. This is something like cascade.